example/cumulus-tf/outputs.tf

```diff
@@ -20,6 +20,10 @@ output "distribution_redirect_uri" {
 
 # Workflow reporting SNS topics
```

Comment:
With this change, this is no longer exactly a _Workflow_ reporting SNS topics section because collections are created differently.

Reply:
Solid point - I can move this out of under the comment. Do you think this is still the right way to expose the SNS topic arn?

Reply:
You can also just remove the `workflow` at the start. This seems correct, yes.